infra/integration-image/check_test.go.bak

```diff
@@ -0,0 +1,216 @@
+package main
```

Comment:
If we duplicating/modifying docker's code like this it needs to be very explicit - note where it comes from and why it's modified as it is.